src/Header/Header.scss

```diff
@@ -171,6 +177,12 @@ header {
         }
       }
 
+      @media (hover: none) {
```

Comment:
I've realised that even though most tablets are big enough to fit the desktop menu, they don't have hover functionality, so we shouldn't show the "arrow down" icon,

Reply:
oh ok.... what about if they tap it, then it just opens up and then then they make their selection or tap somewhere else in order to close it?

Reply:
definitely out of scope for this PR